pkgs/development/python-modules/aiohttp/default.nix

```diff
@@ -1,38 +1,39 @@
 { lib
 , buildPythonPackage
 , fetchPypi
+, fetchpatch
```

Comment:
Can you drop this?

Reply:
Yes, I can. That's left over from 3.0.0.